src/LinuxTracing/PerfEvent.h

```diff
@@ -140,7 +140,7 @@ struct UprobesWithStackPerfEventData {
   uint64_t stream_id;
   pid_t pid;
   pid_t tid;
-  perf_event_sample_regs_user_sp regs;
+  uint64_t sp;
```

Comment:
so that one already did not contained the abi anymore, right?

Reply:
It had the abi but it never used it. It was only assigned and never checked.